packages/lesswrong/components/posts/EALargePostsItem.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useRef } from "react";
+import { Components, registerComponent } from "../../lib/vulcan-lib";
+import { useHover } from "../common/withHover";
+import { postGetPageUrl } from "../../lib/collections/posts/helpers";
+import { siteImageSetting } from "../vulcan-core/App";
+import { AnalyticsContext } from "../../lib/analyticsEvents";
+import { Link } from "../../lib/reactRouterWrapper";
+import { InteractionWrapper } from "../common/useClickableCell";
+import moment from "moment";
+import classNames from "classnames";
+
+const styles = (theme: ThemeType) => ({
+  postListItem: {
+    display: "flex",
+    width: "100%",
+    background: theme.palette.grey[0],
+    border: `1px solid ${theme.palette.grey[100]}`,
```

Comment:
I added this border and background color to match the regular EA posts item which this is now placed next to.